lib/zewo/version.rb

```diff
@@ -1,3 +1,3 @@
 module Zewo
-  VERSION = "0.2.9"
+  VERSION = "0.2.9".freeze
```

Comment:
Indeed, it makes an object immutable like the VERSION constant (everything in Ruby is an object). For example:

~ $ irb
irb(main):001:0> a = "test"
=> "test"
irb(main):002:0> a << "change"
=> "testchange"
irb(main):003:0> a.freeze
=> "testchange"
irb(main):004:0> a << "change"
RuntimeError: can't modify frozen String
	from (irb):4
	from /Users/larssmit/.rbenv/versions/2.3.0/bin/irb:11:in `<main>'
irb(main):005:0>